src/containers/shared/components/Transaction/OfferCreate/index.ts

```diff
@@ -0,0 +1,4 @@
+export { OfferCreateSimple as Simple } from './OfferCreateSimple';
```

Comment:
@mvadari I didn't do a full rename but I simplified they way they are registered.  I think having the longer names will help people navigate the code easier. Otherwise there are going to be 30 files named Simple and components named Simple.

This removes some of the boilerplate.

Reply:
Actually going ahead with a full rename.